tabUtil/tabhttp.py

```diff
@@ -131,11 +133,25 @@ def export_view(configs, view, format, logger):
         protocol = u'https'
     else:
         protocol = u'http'
+        
```

Comment:
That was on my list, I'd particularly like to enable users to control the
image size. My plan was to add code to check for the parameters that
VizAlerts cares about (like :refresh and :size) and use the user-entered
code as an override to VizAlerts.

Jonathan

On Thu, Mar 3, 2016 at 11:16 AM, Matt <notifications@github.com> wrote:

> In tabUtil/tabhttp.py
> <https://github.com/tableau/VizAlerts/pull/14#discussion_r54903028>:
>
> > @@ -131,11 +133,25 @@ def export_view(configs, view, format, logger):
> >          protocol = u'https'
> >      else:
> >          protocol = u'http'
> > +
>
> It's possible that someone could submit a view_url_suffix value that
> already includes the embed URL parameters, e.g.:
>
> myworkbook/myview?team=avengers&:refresh=y&:size=50,50
>
> Might we get some unpredictable behavior? I haven't messed with many
> different combinations, but I could see some confusing stuff if we allow
> the author to put whatever embed parameters in there along with our
> own...what do you think?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/tableau/VizAlerts/pull/14/files#r54903028>.
>